Validation/MuonGEMDigis/plugins/GEMStripDigiValidation.cc

```diff
@@ -82,21 +84,28 @@ void GEMStripDigiValidation::bookHistograms(DQMStore::IBooker& booker,
       Int_t station_id = station->station();
       ME2IdsKey key2{region_id, station_id};
 
+      me_occ_pid_[key2] = bookPIDHist(booker, key2, "sim_occ_pid", "Particle population");
+
       if (detail_plot_) {
         me_detail_total_strip_[key2] =
             bookHist1D(booker, key2, "total_strips_per_event", "Number of strip digs per event", 50, -0.5, 99.5);
 
         me_detail_occ_det_[key2] = bookDetectorOccupancy(booker, key2, station, "strip", "Strip Digi");
 
         me_detail_strip_occ_det_[key2] =
-            bookDetectorOccupancy(booker, key2, station, "matched_strip", "Matched Strip Digi");
+            bookDetectorOccupancy(booker, key2, station, "sim_matched_strip", "Matched Strip Digi");
       }
 
       const auto& superChamberVec = station->superChambers();
       if (superChamberVec.empty() || superChamberVec[0] == nullptr) {
         edm::LogError(kLogCategory_) << "Super chambers missing or null for region = " << region_id
                                      << " and station = " << station_id;
       } else {
+        for (const auto& etaPart : superChamberVec[0]->chambers()[0]->etaPartitions()) {
+          Int_t ieta = etaPart->id().ieta();
+          ME3IdsKey key{region_id, station_id, ieta};
+          me_occ_pid_eta_[key] = bookPIDHist(booker, key2, ieta, "sim_occ_pid", "Particle population");
```

Comment:
I am somehow confused by key*, shouldn't this booker have key as argument onstead of key2?

Reply:
Usually we don't skip the layer part. But several plots are need to skip them.
In this reason, I made a special function for it to keep original form of the `bookPIDHist`.